.gitignore

```diff
@@ -0,0 +1,7 @@
+soloproj/*
+.cache/
+__pycache__/
+allocator.pyc
+people.pyc
+room.pyc
```

Comment:
Instead of ignoring each of the by writing `<file_name>.pyc` files just do `*.pyc`